src/components/Admin/AdminSearchForm.jsx

```diff
@@ -47,8 +47,9 @@ class AdminSearchForm extends React.Component {
       tableData,
       searchParams: { searchCourseQuery, searchDateQuery, searchQuery },
     } = this.props;
-    const courseTitles = new Set(tableData.map(en => en.course_title).sort());
-    const courseDates = new Set(tableData.map(en => en.course_start).sort().reverse());
+    const courseTitles = Array.from(new Set(tableData.map(en => en.course_title).sort()));
```

Comment:
We use `.map` below, which, uh, isn't a Set function. We're unclear on how this ever worked. Maybe a different transpilation of Set used an Array.